cmd/operator-sdk/alpha/olm/up.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olm
+
+import (
+	"fmt"
+	"log"
+
+	olmoperator "github.com/operator-framework/operator-sdk/internal/olm/operator"
+
+	"github.com/spf13/cobra"
+)
+
+func NewUpCmd() *cobra.Command {
+	c := &olmoperator.OLMCmd{}
+	cmd := &cobra.Command{
+		Use:   "up",
+		Short: "Launches the operator with Operator Lifecycle Manager",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return fmt.Errorf("command %q requires exactly one argument", cmd.CommandPath())
+			}
+			c.ManifestsDir = args[0]
+			if err := c.Up(); err != nil {
+				log.Fatalf("Failed to install operator: %v", err)
```

Comment:
In the log we have install, so wdyt about the command be olm install instead of up?